soc/riscv/andes_v5/ae350/Kconfig.soc

```diff
@@ -105,16 +106,8 @@ config SOC_ANDES_V5_PMA_REGION_MIN_ALIGN_AND_SIZE
 		Minimum size (and alignment) of an PMA region. Use this symbol
 		to guarantee minimum size and alignment of PMA regions.
 
-# Workaround for not being able to have commas in macro arguments
-DT_ANDESTECH_L2C := andestech,l2c
-
-config SOC_ANDES_V5_L2C
-	bool
-	default $(dt_compat_enabled,$(DT_ANDESTECH_L2C))
-
```

Comment:
It might make more sense to mark this as deprecated instead of immediately removing it, even if it does nothing.

So remove the default line, add `select DEPRECATED`.

That should allow for downstream users to use the same Kconfig between adjacent Zephyr releases, and they should get a warning in CI about using deprecated options.

Reply:
+1 on marking this as deprecated

Reply:
Thanks! I'll add `select DEPRECATED`.